client/tracing/src/lib.rs

```diff
@@ -78,72 +77,104 @@ pub struct SpanDatum {
 	pub line: u32,
 	pub start_time: Instant,
 	pub overall_time: Duration,
-	pub values: Visitor,
+	pub values: Values,
 }
 
 /// Holds associated values for a tracing span
-#[derive(Clone, Debug)]
-pub struct Visitor(FxHashMap<String, String>);
+#[derive(Default, Clone, Debug)]
+pub struct Values {
+	pub bool_values: FxHashMap<String, bool>,
```

Comment:
for consistency pub fields should have doc.

Reply:
thanks, missed that